src/Form/BasicSetup.php

```diff
@@ -205,7 +211,7 @@ public function buildForm(array $form, FormStateInterface $form_state, User $use
           '#type' => 'submit',
           '#value' => t('Cancel'),
           '#limit_validation_errors' => array(),
-          '#submit' => array('tfa_basic_setup_form_submit'),
+          '#submit' => array('::submitForm'),
```

Comment:
I was going to make a separate function but submitForm was already checking for cancel button so i went with that.
Will take that approach now :)